data/json/monstergroups/resort.json

```diff
@@ -0,0 +1,67 @@
+[
+  {
+    "name": "GROUP_RESORT_DANCERS",
+    "type": "monstergroup",
+    "default": "mon_zombie_resort_dancer",
+    "monsters": [
+      { "monster": "mon_zombie_resort_dancer", "freq": 100, "cost_multiplier": 8, "pack_size": [ 6, 10 ] },
+      { "monster": "mon_zombie_resort_bouncer", "freq": 75, "cost_multiplier": 4, "pack_size": [ 1, 3 ] },
+      { "monster": "mon_zombie", "freq": 50, "cost_multiplier": 10, "pack_size": [ 2, 10 ] }
+    ]
+  },
+  {
+    "type": "monstergroup",
+    "name": "GROUP_RESORT_RAID",
+    "default": "mon_zombie_cop",
+    "monsters": [
+      { "monster": "mon_zombie_cop", "freq": 280, "cost_multiplier": 2, "pack_size": [ 3, 8 ] },
+      { "monster": "mon_dog_zombie_cop", "freq": 50, "cost_multiplier": 1, "pack_size": [ 1, 2 ] }
+    ]
+  },
+  {
+    "name": "GROUP_RESORT_POOL",
```

Comment:
How is this different from the existing pool monster group?

Reply:
It has no kids. I suppose considering rather adult idea of the place it's important.

Reply:
I'd suggest then that rather than making a monstergroup that's dedicated to this location, you go to the zombies monstergroups and make this GROUP_POOL_NOKIDS

Reply:
If that won't be seen as going out of changes limit for this PR, I'll do it.

Reply:
Nope, you can definitely edit other files besides the scope of the PR. In fact as much as possible we'd prefer you not to make a bunch of dedicated monstergroups and things like that, because it just adds more clutter to maintain them.